packages/x-data-grid/src/DataGrid/DataGrid.tsx

```diff
@@ -68,7 +68,7 @@ interface DataGridComponent {
  * API:
  * - [DataGrid API](https://mui.com/x/api/data-grid/data-grid/)
  */
-export const DataGrid = React.memo(DataGridRaw) as DataGridComponent;
+export const DataGrid = React.memo(forwardRef(DataGridRaw)) as DataGridComponent;
```

Comment:
Moved the forwardRef inside the memo to prevent the component name being `DataGridRaw`, though I don't know if this is ok. I would expect so, since the entire thing is memoized anyways.

https://app.circleci.com/pipelines/github/mui/mui-x/92158/workflows/efcac4b3-6861-4171-b28c-b67d80af2cbb/jobs/538493